src/const.js

```diff
@@ -1,6 +1,10 @@
 export const DEFLATE_COMPRESSION = 8;
 export const NO_COMPRESSION = 0;
 
+export const SEVERITY_ERROR = 2;
```

Comment:
Could this not use VALIDATION_* instead. Seems it might get confusing to have uses of similar but different constants. I'm easily confused :)